deep-learning/README.md

```diff
@@ -1,6 +1,9 @@
-# Deep Learning Resources
+# Machine Learning Resources
```

Comment:
Neural Networks != ML 😬. That's why I had put Deep Learning Resources. Also folder name is deep-learning.

Reply:
Change folder name to ML. i m on windows

Reply:
No, these are DL resources specifically. Not ML resources. ML means traditional ML algos, not just neural networks. So, maybe make a new folder for that.